actindo-grid.html

```diff
@@ -787,13 +803,20 @@
                         value: function(){return [];}
                     },
 
-                     /**
+                    /**
                       * List of the optional column. The hidden column will not be in this list
-                      */
-                     _serializeOptionals: {
+                     */
+                    _serializeOptionals: {
                         type: Array,
                         value: function(){return [];}
-                     }
+                    },
+
+                    /**
+                      * internal property, the title  will show on headline
+                     */
```

Comment:
I'd call this property "title" as it is more descriptive.
Please remove "internal property" from the description, this is meant to be used from the outside